microsetta_public_api/api/microsetta_public_api.yml

```diff
@@ -256,13 +353,84 @@ paths:
                                 - 8.25
                                 - 9.01
                                 - 9.04
-
         '404':
           $ref: '#/components/responses/404NotFound'
         '400':
           description: >
             Either `summary_statistics`, `return_raw`, or both are required to be true.
 
+  '/diversity/beta/pcoa/contains/{beta_metric}/{named_sample_set}/{sample_id}':
+    get:
+      operationId: microsetta_public_api.api.diversity.beta.pcoa_contains
+      tags:
+        - Beta Diversity
+      summary: Get whether the given `sample_id` is contained in the `named_sample_set`
+      description: Get whether the given `sample_id` is contained in the `named_sample_set`
+      parameters:
+        - in: path
+          name: beta_metric
+          description: Plot the data for samples using this metric
+          schema:
+            $ref: '#/components/schemas/betaMetric'
+          required: true
+        - in: path
+          name: named_sample_set
+          description: The named sample set that should be plotted
+          schema:
+            type: string
+            example: 'habitat-type'
+          required: true
+        - in: path
+          name: sample_id
+          schema:
+            $ref: '#/components/schemas/sampleId'
+          required: true
+      responses:
+        '200':
+          description: a `boolean`, which indicates if `sample_id` is contained in `named_sample_set`
+          content:
+            application/json:
+              schema:
+                type: boolean
+        '404':
+          $ref: '#/components/responses/404NotFound'
+
+  '/plotting/diversity/beta/pcoa/{beta_metric}/{named_sample_set}':
+    get:
+      operationId: microsetta_public_api.api.plotting.plot_beta
```

Comment:
I might vote for /plotting/diversity/beta/{beta_metric}/{named_sample_set}/pcoa, since you compute the sample set perhaps supercedes (e.g., you may do some other computation on a named sample set), but sure.

Reply:
Done.